### ISSUE TITLE
multi-row_tabs.css issue

### ISSUE THREAD

**Issue by multiadres** [MrOtherGuy/firefox-csshacks#414]
After the recent upgrade of Firefox to version 131.0, the solution stopped working. I ran the whole procedure from scratch and all settings are correct, but the method does not work. It is interesting that the tab bar on the right side displays the scrollbar but only one row of tabs is displayed.
![2024-10-04_10-00-22](https://github.com/user-attachments/assets/81777a64-6e8f-41c6-aa31-8986270d9d6e)

**Comment by MrOtherGuy** [comment]
Hmm, you are using Windows, right? multi-row_tabs.css should work just fine with Firefox 131 on Windows.

**Comment by dkrasnop** [comment]
It stopped working for me either

**Comment by MrOtherGuy** [comment]
Could you confirm you are using up-to-date version of multi-row_tabs.css?

**Comment by dkrasnop** [comment]
I do not use up-to-date version. I am using the one I used in 130

**Comment by dkrasnop** [comment]
but multiadres user updated the file and it still did not work for him

**Comment by multiadres** [comment]
> Could you confirm you are using up-to-date version of multi-row_tabs.css?

I loaded your latest version multi-row_tabs.css and everything started working. Thank you very much for your quick response and efficient fix.

**Comment by dkrasnop** [comment]
@multiadres You said " I ran the whole procedure from scratch and all settings are correct, but the method does not work. " So you did not do it from scratch as you did not use the file from github. I wasted all day relaying on your comment. Please be careful next time you claim something.

**Comment by multiadres** [comment]
@dkrasnop I apologize if there was a misunderstanding, but this morning at 9:00 a.m. I copied the CSS, next to which it said it was from two months ago. And it didn't work. It wasn't until the evening that I copied that CSS next to which it said it was uploaded to github around 2 p.m. And it worked. Sorry to have wasted your day, it was not my intention and I really tried to do everything in a correct way because the lack of multirow-tabs made my work very difficult.
I want to thank the creator of this solution for the job he is doing because it makes the work very easy. This applies to the rest of the CSS as well.
